api/room/schema.py

```diff
@@ -90,9 +90,21 @@ def mutate(self, info, office_id, **kwargs):
         admin_roles.create_rooms_update_delete_office(office_id)
         query = Room.get_query(info)
         exact_query = room_join_office(query)
+
+        # prevent analytics data being corrupted by
+        # checking that rooms created have unique calendar ids
+        room_with_same_calender_id = query.filter(
+            RoomModel.state == "active",
+            RoomModel.calendar_id == kwargs['calendar_id']
+        )
+        if room_with_same_calender_id.count() > 0:
```

Comment:
Why do we have to count, can't we just check without counting?

Reply:
An object is returned whether it is a positive or negative result as regards the if-check.
.count is, however, able to obtain a negative result on the query result that is run in this case and hence it is employed here.